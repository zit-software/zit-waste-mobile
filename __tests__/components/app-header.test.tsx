import renderer from 'react-test-renderer';

import AppHeader from '../../components/app-header';

describe('<AppHeader />', () => {
	it('should be render', () => {
		const tree = renderer
			.create(<AppHeader />)
			.toJSON() as renderer.ReactTestRendererJSON;

		expect(tree).toMatchSnapshot();
	});
});
