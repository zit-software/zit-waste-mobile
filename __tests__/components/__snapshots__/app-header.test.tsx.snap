// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppHeader /> should be render 1`] = `
<View
  style={
    {
      "paddingTop": 10,
    }
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "flexDirection": "row",
        "gap": 15,
      }
    }
  >
    <Image
      source={1}
      style={
        {
          "height": 45,
          "objectFit": "contain",
          "width": 45,
        }
      }
    />
    <Text
      style={
        {
          "color": "#1455B6",
          "fontFamily": "Montserrat_800ExtraBold",
          "fontSize": 24,
        }
      }
    >
      Zit Waste
    </Text>
  </View>
</View>
`;
